Source: https://gitweb.gentoo.org/repo/gentoo.git/plain/dev-libs/libtsm/files/libtsm-clang16-static_assert-fix.patch

Fix -Wimplicit-function-declaration error with gcc 14.

```
/home/buildozer/aports/testing/libtsm/src/libtsm-4.0.2/src/shared/shl-macro.h:167:30: error: implicit declaration of function 'static_assert' [-Wimplicit-function-declaration]
  167 | #define shl_assert_cc(_expr) static_assert(_expr, #_expr)
      |                              ^~~~~~~~~~~~~
```

--
Bug: https://bugs.gentoo.org/895052
--- a/cmake/CompileOptions.cmake
+++ b/cmake/CompileOptions.cmake
@@ -1,5 +1,5 @@
 # Set compiler flags
-set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_STANDARD 11)
 set(CMAKE_C_STANDARD_REQUIRED ON)
 
 # analogous to AC_USE_SYSTEM_EXTENSIONS in configure.ac
